pkg/provisioner/ironic/testserver/ironic.go

```diff
@@ -25,12 +25,25 @@ type IronicMock struct {
 
 // NewIronic builds an ironic mock server
 func NewIronic(t *testing.T) *IronicMock {
+
 	return &IronicMock{
 		MockServer:   New(t, "ironic"),
 		CreatedNodes: nil,
 	}
 }
 
+// WithDefaultResponses sets a valid answer for all the API calls
```

Comment:
Maybe this could call Ready() to set up those responses as well?

Reply:
Sounds reasonable